### DISCUSSION TITLE
rootless docker

### DISCUSSION THREAD

**Discussion by skyzuma** [nicolargo/glances#2071] [General]
hello, i switched to rootless docker and now i cant use glances (no docker networks / container - installed on arm64v8) ... i used glances for years ... its possible to use glances with rootless docker? any settings?

**Comment by amzon-ex** [comment]
Hi, I would like to know this as well! Currently, no container info is displayed and the following error pops up in the log file:
`2024-11-09 02:37:18,630 -- ERROR -- containers (Docker) plugin - Can't connect to Docker (Error while fetching server API version: ('Connection aborted.', PermissionError(13, 'Permission denied')))`
